skuba-update/skuba_update/skuba-update.timer

```diff
@@ -3,9 +3,9 @@ Description=Daily update of the system
 After=network-online.target local-fs.target
 
 [Timer]
+OnActiveSec=0
 OnCalendar=daily
 AccuracySec=1m
-RandomizedDelaySec=2h
```

Comment:
Yes, please saw comment https://github.com/SUSE/skuba/pull/894#pullrequestreview-331765706. [1]

We want to do

> skuba when finished bootstrapping/joining calls skuba-update by using systemd, so it will annotate the node immediately after. This will also make skuba-update annotate the nodes immediately with no updates, while at the moment they are <none> until skuba-update is run the first time by the timer.

Therefore, I remove this line.

Could you kindly provide me the reason why not do base OS patches at the same time?

Reply:
When the patches requires service restart or reboot, update all nodes at the same time would risk cluster downtime.

Reply:
For the node reboot is handled by `kured` and it guarantees that only one node reboots.

For the service restart, AFAIK, it does not include the Kubernetes controller plan updates, with the OS host rpm package patch update. IMO, service restart should not influence the Kubernetes cluster to have the downtime

Reply:
Reverted.

Do skuba-update once when node bootstrap/join with `systemctl start --no-block skuba-update`. Then, this keeps the original behavior that not all nodes at the same time are trying to update base OS patches.